src/QMCDrivers/QMCDriver.h

```diff
@@ -399,6 +400,8 @@ class QMCDriver: public QMCTraits, public MPIObjectBase
   std::string getRotationName( std::string RootName);
   std::string getLastRotationName( std::string RootName);
 
+  NewTimer *checkpointTimer;
+
```

Comment:
I saw above
std::vector<NewTimer*> myTimers;
It is it beneficial to engage the above one?
Probably this will introduce some entanglement with the enumeration in the VMC/DMC drivers.

Reply:
Probably not, since it's not used in this class.
What should probably be done is to move that declaration of myTimers to the individual classes that use it.

Reply:
I agree.